### PR TITLE
update config syntax in registry.md

### PR DESCRIPTION
**Pull request by cofyc**

### CODE REVIEW

**Comment by cofyc** [comment]
cc @Random-Liu @mikebrow

**Comment by mikebrow** [comment]
@cofyc thanks for the PR! Could you sign the commit? 

git commit -s --amend
git push -f origin update-config-syntax

**Comment by cofyc** [comment]
@mikebrow done

File: docs/registry.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ With containerd, `docker.io` is the default image registry. You can also set up
 
 To configure image registries create/modify the `/etc/containerd/config.toml` as follows:
 ```toml
```
Comment (AkihiroSuda) [inline docs/registry.md]: Can we add `version = 2` header here?

Comment (AkihiroSuda) [reply docs/registry.md]: Also, we should add a note about the difference between v1 and v2

Comment (Random-Liu) [reply docs/registry.md]: Agree

Comment (cofyc) [reply docs/registry.md]: addressed in 51913a5dc3a4c7398e00e5241657a4b6a440da0d, please check

File: docs/registry.md [hunk 2]
```diff
@@ -1,20 +1,27 @@
 # Configure Image Registry
 This document describes the method to configure the image registry for `containerd` for use with the `cri` plugin.
 
+NOTE: The configuration syntax used in this doc is in version 2 which is the
+default since containerd 1.13.0. If your configuration is still in version 1,
```
Comment (AkihiroSuda) [inline docs/registry.md]: "default since containerd 1.13.0" ->  "recommended since containerd 1.3" ?

Comment (cofyc) [reply docs/registry.md]: fixed, you're right, the default version (if not set) is still the version 1.

File: docs/registry.md [hunk 3]
```diff
@@ -1,20 +1,27 @@
 # Configure Image Registry
 This document describes the method to configure the image registry for `containerd` for use with the `cri` plugin.
 
+NOTE: The configuration syntax used in this doc is in version 2 which is the
+recommended since containerd 1.13.0. If your configuration is still in version 1,
```
Comment (AkihiroSuda) [inline docs/registry.md]: 1.13.0 -> 1.3

Comment (cofyc) [reply docs/registry.md]: done

Comment (AkihiroSuda) [reply docs/registry.md]: Seems not fixed yet. "containerd 1.13" was never released out.

Comment (cofyc) [reply docs/registry.md]: fixed

sorry about my stupid mistake...

**Comment by mikebrow** [comment]
@cofyc sry, needs a rebase

**Pull request merged**